privacy_consent/views/privacy_activity.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 Tecnativa - Jairo Llopis
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<data>
+
+    <record id="activity_form" model="ir.ui.view">
+        <field name="name">Add consent fields</field>
+        <field name="model">privacy.activity</field>
+        <field name="inherit_id" ref="privacy.activity_form"/>
+        <field name="arch" type="xml">
+            <div name="button_box" position="inside">
+                <button
+                    attrs='{"invisible": [("consent_required", "=", False)]}'
+                    class="oe_stat_button"
+                    context='{"search_default_activity_id": active_id}'
+                    icon="fa-users"
+                    name="%(consent_action)d"
+                    type="action"
+                >
+                    <field
+                        name="consent_ids_count"
+                        widget="statinfo"
+                    />
+                </button>
+            </div>
+
+            <notebook name="advanced" position="inside">
+                <page string="Consent" name="consent">
```

Comment:
Call the elements with the node name: `page_consent`, or the possibility to choke with other names will be very high

Reply:
I know, but it's for consistency and to not need the element, only the name.

Reply:
Consistency on what? I have never seen such pattern... :roll_eyes:

Reply:
I have seen it across Odoo and OCA modules, but it's not a written rule.

Reply:
To be consistent, I'd have to rename `<div name="button_box">` to `<div name="div_button_box">`, `<group name="basic">` to `<group name="group_basic">`, etc? It doesn't make much sense AFAICS. There's no problem on using the element name in an xpath if you really need it, the same way as there's no problem on using only the name if you don't really need the element.

Reply:
OK, do it as you want. FTR, I do it always on groups and pages. Button box itself doesn't need to specify that it's a div, but you can have a field, a group or a page with the same `name` attribute.